Exiled.API/Features/Lift.cs

```diff
@@ -264,106 +264,6 @@ public override Quaternion Rotation
-
-        /// <summary>
-        /// Temporary locks all elevators corresponding to the given types.
-        /// </summary>
-        /// <param name="types">The doors to affect.</param>
-        /// <param name="duration">The duration of the lockdown.</param>
-        /// <param name="lockReason">The specified <see cref="DoorLockReason"/>.</param>
-        public static void LockAll(IEnumerable<ElevatorType> types, float duration, DoorLockReason lockReason = DoorLockReason.Isolation) => types.ForEach(t => Lock(t, lockReason));
-
-        /// <summary>
-        /// Unlocks all lifts in the facility.
-        /// </summary>
-        public static void UnlockAll() => List.ForEach(lift => lift.Unlock());
-
-        /// <summary>
-        /// Unlocks all lifts in the facility.
-        /// </summary>
-        /// <param name="type">The zones to affect.</param>
-        public static void UnlockAll(ZoneType type) => List.ForEach(lift => lift.Doors.Where(door => door.Zone == type).ForEach(door => door.Unlock()));
-
-        /// <summary>
-        /// Unlocks all lifts in the facility.
-        /// </summary>
-        /// <param name="types">The zones to affect.</param>
-        public static void UnlockAll(params ZoneType[] types) => List.ForEach(lift => lift.Doors.Where(door => types.Contains(door.Zone)).ForEach(door => door.Unlock()));
-
-        /// <summary>
-        /// Unlocks all lifts in the facility.
-        /// </summary>
-        /// <param name="types">The zones to affect.</param>
-        public static void UnlockAll(IEnumerable<ZoneType> types) => List.ForEach(lift => lift.Doors.Where(door => types.Contains(door.Zone)).ForEach(door => door.Unlock()));
-
-        /// <summary>
-        /// Unlocks all lifts in the facility.
-        /// </summary>
-        /// <param name="types">The types to affect.</param>
-        public static void UnlockAll(params ElevatorType[] types) => Get(types).ForEach(lift => lift.Unlock());
-
-        /// <summary>
-        /// Unlocks all lifts in the facility.
-        /// </summary>
-        /// <param name="types">The types to affect.</param>
-        public static void UnlockAll(IEnumerable<ElevatorType> types) => Get(types).ForEach(lift => lift.Unlock());
-
-        /// <summary>
-        /// Tries to melt a <see cref="Player"/>.
-        /// </summary>
-        /// <param name="player">The <see cref="Player"/> to melt.</param>
-        /// <returns><see langword="true"/> if the player was melted successfully; otherwise, <see langword="false"/>.</returns>
-        /// <seealso cref="Player.EnableEffect(EffectType, float, bool)"/>
-        public static bool TryMeltPlayer(Player player)
-        {
-            if (player.Position.y is >= 200 or <= -200)
-                return false;
-
-            player.EnableEffect(EffectType.Decontaminating);
-
-            return true;
-        }
```

Comment:
Why is this removed ?
because now when it's Decontamination NW is constantly checking if it's give Decontaminating Effect so it's now useless to have this method